etc/aqd.conf.defaults

```diff
@@ -155,6 +155,8 @@ plenarydir = %(cfgdir)s/plenary
 service = %(user)s
 keytab = /var/spool/keytabs/%(service)s
 installfe_user = %(user)s
+# Uncomment and define installfe_cdburl if you want Aquilon to manage cdburl
+#installfe_cdburl = Aquilon profiles URL
```

Comment:
The point of has value is that it allows to keep the empty variable namespace without a need to comment it out. Suggest to do this like it:
# Define installfe_cdburl if you want Aquilon to manage cdburl
installfe_cdburl =

Reply:
It was what I did initially but I got an error with an empty value... I should probably retest it but as I'm leaving tomorrow evening for one month, I am not sure I'll be able to do it before... Will do my best!

Reply:
Please share the error, but i tested the has_value plenty of times.. weird...